count-on-click/style.css

```diff
@@ -0,0 +1,48 @@
+* {
+  background: rgb(227, 210, 210);
+  margin: 0;
+  padding: 0;
+  font-weight: bolder;
+  color: rgba(240, 33, 33, 0.519);
+  user-select: none;
+  transition: 100ms all ease-in-out;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  height: 100vh;
+}
+.clicker {
+  display: flex;
+  transform: rotate(-90deg);
+  width: 50px;
+  height: 50px;
+  cursor: pointer;
+  align-items: center;
+  justify-content: center;
+  font-size: 40px;
+}
+
+.cntr {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: 8px;
+  background: rgba(255, 0, 0, 0.848);
+  border-radius: 15px;
+  color: white;
+  min-width: 30px;
+  font-size: 30px;
+}
```

Comment:
Counter doesn't seem to have equal length on all 4 sides. It should be a square as shown in preview.
![telegram-cloud-photo-size-5-6064640379577282667-x](https://user-images.githubusercontent.com/51869689/186597075-575360d6-d4c7-41f8-bd22-9789268f5842.jpg)